
{
    "error_translatable": "This username is already taken"
    "validation_messages": {
        "username": {
            "usernameAlreadyTaken": "User already exists"
        }
    },
    "type": 102,
    "title": "This username is already taken",
    "status": 422,
    "detail": "error.api.username.alreadytaken"
}

